apps/explorer/src/components/NetworkSelector/index.tsx

```diff
@@ -63,7 +63,7 @@ export const NetworkSelector: React.FC<networkSelectorProps> = ({ networkId }) =
             const url = shouldPreservePath ? `${network.urlAlias}/${pathSuffix || ''}` : network.urlAlias
 
             return (
-              <Option to={url} color={network.color} key={itemNetworkId}>
+              <Option to={'../' + url} color={network.color} key={itemNetworkId}>
```

Comment:
Relative links now need to be prefixed with `../`

https://reactrouter.com/upgrading/v6#v7_relativesplatpath